docs/contributing/gsoc/2021/index.rst

```diff
@@ -115,7 +110,7 @@ Deadlines
 See the milestones prefixed with "GSoC 2021"
 https://github.com/intel/dffml/milestones?direction=desc&sort=due_date&state=open
 
-Please see the :doc:`/contributing/gsoc/rubric` page for the project
+Please see the: doc:`/contributing/gsoc/rubric` page for the project
```

Comment:
`:doc:` actually makes it a link to the path after it in the docs, we don't need to move the ":" ,  otherwise it would link to the desired page , you'd need to revert this change :)

Reply:
okay got it

Reply:
oh, okay got it